scripts_checker.py

```diff
@@ -0,0 +1,61 @@
+import win32wnet
+import os
+import sys
+import git
+import concurrent.futures
+# noinspection PyUnresolvedReferences
+from win32wnet import error as PywintypesError
+from util.channel_access import ChannelAccessUtils
+
+
+inst_hostnames = [inst['hostName'] for inst in ChannelAccessUtils().get_inst_list()]
+different_head, multiple_repos, cannot_connect = [], [], []
+
+
+def check_inst_scripts(hostname):
+    scripts_path = f'\\\\{hostname}\\c$\\Instrument\\scripts'
+
+    # Connect to the instrument shared network resource
+    username = f'{hostname}\\{os.environ.get("USER")}'
+    password = os.environ.get("PASS")
+    try:
+        # dwType, lpLocalName, lpRemoteName[, lpProviderName, Username, Password, flags]
+        # noinspection PyArgumentList
+        win32wnet.WNetAddConnection2(0, None, scripts_path, None, username, password)
+    except PywintypesError as e:
+        sys.stderr.write(f'Error {e.winerror} connecting to {hostname}: {e.strerror}\n')
+        cannot_connect.append(hostname)
+        return
+
+    repo = git.Repo(path=scripts_path)
+
+    if len(repo.remotes) > 1:
+        print(f'WARNING: {hostname} has multiple remote repositories: \n{repo.git.remote("-v")}\n')
+        multiple_repos.append(hostname)
+
+    current_head = repo.head.commit
+    master_head = repo.heads.master.commit
```

Comment:
This is comparing to local master, we want to compare to the remote master. i.e. they should both be being compared to https://github.com/ISISNeutronMuon/InstrumentScripts/commit/ebcb09c22962fa7b326a3282b4c653def7144c20